doc/api/crypto.md

```diff
@@ -1883,6 +1886,23 @@ the corresponding digest algorithm. This does not work for all signature
 algorithms, such as `'ecdsa-with-SHA256'`, so it is best to always use digest
 algorithm names.
 
+### crypto.isKeyObject(value)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `value` {any}
+* Returns: {boolean}
+
+Returns `true` if the value is a [`KeyObject`] instance.
```

Comment:
Should we add a note that this does not work with other contexts / realms?

Reply:
I'm afraid I do not follow :(

Reply:
The function uses `obj instanceof KeyObject` internally. This does not work in case some code is run in e.g. the `vm`.

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof#instanceof_and_multiple_context_(e.g._frames_or_windows) for further details.

Reply:
Can you suggest the language? I'm happy to add it.

Reply:
Maybe:

```
Returns `true` if the value is a [`KeyObject`][] instance from the [current execution context][].

// ...

[current execution context]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof#instanceof_and_multiple_context_(e.g._frames_or_windows)
```

But this might be confusing to others. So it's maybe something we do not have to add?